lang/python-evdev/Makefile

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright (C) 2016 OpenWrt.org
```

Comment:
Drop this line entirely, or use your own name.  (Unless you really did sign some sort of copyright assignment to openwrt....)